### PR TITLE
"Make a PR" - junkystuff

### PR DESCRIPTION
**Pull request by alexkar598**
# Document the changes in your pull request

Adds a vital PR

# Wiki Documentation

None of your bloody business

# Changelog

:cl:  
rscadd: PR
/:cl:

### CODE REVIEW

**Comment by Junkyshell** [comment]
thanks

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+pr
```
Comment (ToasterBiome) [inline README.md]: remove this

**Review by ToasterBiome — CHANGES_REQUESTED** [review]

**Comment by Cartlord** [comment]
Please actually edit the template.

**Comment by ynot01** [comment]
can-we-ban-this-guy

**Comment by alexkar598** [comment]
@Cartlord Is this to your satisfaction my very good sir?

**Comment by Hopekz** [comment]
OH MY GOD WHO WILL STOP THIS MAN

**Comment by maxion12345** [comment]
i ded pr + ratio + you fell off + who asked + L+ youngboy made a better covid vaccine 😭 + nobody cares + like this if yogstation is better then tg + loss.jpeg+ sussy + weeb pfp + no opinion + make me admin + obamas last name is president

**Review by ToasterBiome — APPROVED** [review]
code looks good, up to HC as this is a balance change

**Pull request merged**